doc/src/challenges/0003-pointer-arithmentic.md

```diff
@@ -0,0 +1,107 @@
+# Challenge 3: Verifying Raw Pointer Arithmetic Operations
+
+- **Status:** Open
+- **Solution:**
+- **Tracking Issue:** <https://github.com/model-checking/verify-rust-std/issues/21>
+- **Start date:** 24/06/24
+- **End date:** 24/12/10
+
+-------------------
+
+
+## Goal
+
+The goal of this challenge is to verify safety of code that relies on raw pointer arithmetic, and eventual
+raw pointer access.
```

Comment:
I think this needs more precise scoping: which "code" that "relies on raw pointer arithmetic" do you actually want to cover?

Reply:
Can you point me to where exactly this is happening? I read "usage of these different methods within the standard library" and it makes me think this challenge requires verification of the _entire_ standard library.

Reply:
That's the success criteria where we list all the functions that must be annotated and verified.